_packages/native-preview/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "private": true,
+    "name": "@typescript/native-preview",
+    "version": "0.0.0",
+    "license": "Apache-2.0",
+    "author": "Microsoft Corp.",
+    "homepage": "https://www.typescriptlang.org/",
+    "description": "Preview CLI of the native TypeScript compiler port",
+    "keywords": [
+        "TypeScript",
+        "Microsoft",
+        "compiler",
+        "language",
+        "javascript"
+    ],
+    "bugs": {
+        "url": "https://github.com/microsoft/typescript-go/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/typescript-go.git"
+    },
+    "type": "module",
```

Comment:
Doesn't matter much but I would move `type: module` down next to `exports`, or towards the top.

Reply:
Sure, though I usually put "info for humans" on top, followed by "info for computers". (In case you wanted a rationale 😄)